src/background/adblocker.js

```diff
@@ -308,19 +305,11 @@ function injectScriptlets(filters, tabId, frameId, hostname) {
 }
 
 function injectStyles(styles, tabId, frameId) {
-  const target = { tabId };
-
-  if (frameId !== undefined) {
-    target.frameIds = [frameId];
-  } else {
-    target.allFrames = true;
```

Comment:
This is most important part of this PR - we should not inject to all frames, this was just incorrect.